src/rest.ts

```diff
@@ -6,19 +6,25 @@ import {
 } from '@/types';
 import { Dandiset as DandisetMetadata, DandisetContributors, Organization } from '@/types/schema';
 
+if (!process.env.VUE_APP_DANDI_API_ROOT) {
+  throw new Error('Environment variable "VUE_APP_DANDI_API_ROOT" must be set.');
+}
+
 // Ensure contains trailing slash
 const dandiApiRoot = process.env.VUE_APP_DANDI_API_ROOT.endsWith('/')
   ? process.env.VUE_APP_DANDI_API_ROOT
   : `${process.env.VUE_APP_DANDI_API_ROOT}/`;
 
 const client = axios.create({ baseURL: dandiApiRoot });
 
-let oauthClient: OAuthClient|null;
+let oauthClient: OAuthClient|null = null;
```

Comment:
```suggestion
let oauthClient?: OAuthClient = null;
```
I think this is the magic syntax

Reply:
Unfortunately `| null` looks like the only way to do it. https://github.com/microsoft/TypeScript/issues/13321

Reply:
sad :'(